src/pages/careers/sr-open-source-engineer.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { createUseStyles } from 'react-jss';
+
+import { SEO, SitewideFooter, LayoutControl, SitewideHeader } from 'components';
+import Hero from 'components/careers/Hero';
+import Footer from 'components/careers/Footer';
+import Main from 'components/careers/Main';
+import { backstageLink } from 'components/careers/links';
+
+const useStyles = createUseStyles(() => ({
+  sitewideHeaderWrapper: {
+    marginBottom: 40,
+  },
+}));
+
+const ROLE_NAME = 'Senior Open Source Engineer';
```

Comment:
What's the difference between Open Source and open-source? I did a quick Google and can't find anything meaningful.

Reply:
It is incorrect spelling on my part. It should read "Open-Source". The hyphen would appear to be the convention for spelling that word using tools like Grammarly etc. there does not appear to be a consistency of the spelling online.

Reply:
Action : will make it "Open-Source"